src/core/solutions/chung_reynolds_function.cpp

```diff
@@ -0,0 +1,54 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+/* 
+ * File:   chung_reynolds_function.cpp
+ * Author: Alejandro Ruiz Madruga <amadruga at estudiantes.uci.cu>
+ *
+ * Created on 29 de septiembre de 2019, 20:03
+ */
+
+#include <math.h>
+#include <core/solutions/chung_reynolds_function.h>
+
+namespace dnn_opt
+{
+namespace core
+{
+namespace solution
+{
+    chung_reynolds_function* chung_reynolds_function::make(generator* generator,
+    unsigned int size)
+    {
+        auto* result= new chung_reynolds_function(generator, size);
+        result->init();
+        return result;
+    }
+    float chung_reynolds_function::calculate_fitness()
+    {
+        float result=0;
+        float* params=get_params();
+        
+        solution::calculate_fitness();
+        
+        for(int i=0;i<size();i++)
+        {
+            result+=pow(params[i],2);
+        }
+        return pow(result,2);
+    }
+    chung_reynolds_function::chung_reynolds_function(generator* generator, 
+    unsigned int size):solution(generator,size)

```

Comment:
Why this line break??
In the worst case scenario, line breaks in this specification are only allowed in the member initialization section (after the `:`). Please check other files for examples

Reply:
Netbeans has a red line in that position. I imagine that this limited the amount of characters in a code line. thats why a introduce a separator. But i could supress them

Reply:
take a look at: https://github.com/jairodelgado/dnn_opt/blob/master/src/core/solutions/rastrigin.cpp line 36, break the line in that position